### ISSUE TITLE
what about IDL's (like Corba-IDL)? You should name them!!

### ISSUE THREAD

**Issue by gernotstarke** [isaqb-org/curriculum-api#31]
LG-5-4 (Description) does not give any examples of alternative description methods.

At least Corba-IDL had (and, astonishingly still has!) application in industry. 

Would be nice to both name those alternatives AND give references!

**Comment by dret** [comment]
Good point. We have listed alternative approaches in [chapter 3](https://public.isaqb.org/curriculum-api/release-candidate/curriculum-api-de.html#_begriffe_und_konzepte_3) but I agree that these should be listed in chapter 5 as well. I am sure that pretty much every trainer will mention CORBA at some point, but that by now is so ancient that I wouldn't list it in the curriculum. However, more modern approaches such as gRPC, GraphQL, and AsyncAPI definitely should be listed as alternative description methods based on alternative underlying styles and tech stacks.
